src/manifest.json

```diff
@@ -2,14 +2,14 @@
   "manifest_version": 2,
   "name": "X-Original-To Column",
   "description": "Adds a column for the X-Original-To header to message list panel",
-  "version": "0.2.3",
+  "version": "0.3.0",
   "author": "Peter Fabinski",
   "homepage_url": "https://github.com/peterfab9845/original-to-column",
   "applications": {
     "gecko": {
       "id": "original-to-column@peterfab.com",
-      "strict_min_version": "68.0a1",
-      "strict_max_version": "102.*"
+      "strict_min_version": "115.0",
```

Comment:
This doesn't seem right - I tested with 115.9.0 and it does not have the API change yet.
How can I tell which release version includes changeset [ed5caa97f280d2b49d861146e01e98cfc90142e7](https://hg.mozilla.org/comm-central/rev/ed5caa97f280d2b49d861146e01e98cfc90142e7)?

Reply:
It is 115.10: https://bugzilla.mozilla.org/show_bug.cgi?id=1817682#c32